.github/workflows/ci.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
         uses: "ramsey/composer-install@v2"
         with:
           dependency-versions: "${{ matrix.dependencies }}"
+        env:
+          COMPOSER_AUTH: '{"github-oauth": {"github.com": "${{ secrets.GITHUB_TOKEN }}"}}'
```

Comment:
What is this a PAT?

Reply:
I think it comes from Github itself https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

The permissions for forked repos are all `read` https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token